pkg/resourceinterpreter/defaultinterpreter/prune/prune.go

```diff
@@ -14,38 +14,10 @@ import (
 // RemoveIrrelevantField used to remove fields that generated by kube-apiserver and no need(or can't) propagate to
 // member clusters.
 func RemoveIrrelevantField(workload *unstructured.Unstructured) error {
-	// populated by the kubernetes.
-	unstructured.RemoveNestedField(workload.Object, "metadata", "creationTimestamp")
-
-	// populated by the kubernetes.
-	// The kubernetes will set this fields in case of graceful deletion. This field is read-only and can't propagate to
-	// member clusters.
-	unstructured.RemoveNestedField(workload.Object, "metadata", "deletionTimestamp")
-
-	// populated by the kubernetes.
-	// The kubernetes will set this fields in case of graceful deletion. This field is read-only and can't propagate to
-	// member clusters.
-	unstructured.RemoveNestedField(workload.Object, "metadata", "deletionGracePeriodSeconds")
-
-	// populated by the kubernetes.
-	unstructured.RemoveNestedField(workload.Object, "metadata", "generation")
-
-	// This is mostly for internal housekeeping, and users typically shouldn't need to set or understand this field.
-	// Remove this field to keep 'Work' clean and tidy.
-	unstructured.RemoveNestedField(workload.Object, "metadata", "managedFields")
-
-	// populated by the kubernetes.
-	unstructured.RemoveNestedField(workload.Object, "metadata", "resourceVersion")
-
-	// populated by the kubernetes and has been deprecated by kubernetes.
```

Comment:
To be honest, I can't see much benefit from this change. In addition, I hope this kind of comment is still useful and should not be removed.

Reply:
I think so, these comments is useful for me to understand why to remove these fields.

Reply:
Comments can left above getIrrelevantFields func.The benefit is the removal of redundant code,if we want to add more fields to ignore,we have to copy `unstructured.RemoveNestedField(workload.Object, "metadata", "uid")` and paste.